configure

```diff
@@ -676,32 +677,29 @@ for lib in $generated_META $lbytes; do
         -e "s|%%interfaces%%|${if}|g" \
         -e "s|%%findlib_version%%|${version}|g" \
         -e "s|%%natdynlink%%|${natdynlink}|g" \
-        -e "s|%%dynlink_dir%%|${dynlink_dir}|g" \
-        -e "s|%%unix_dir%%|${unix_dir}|g" \
-        -e "s|%%str_dir%%|${str_dir}|g" \
-        site-lib-src/$lib/META.in > site-lib-src/$lib/META
+        -e "s|%%dynlink_dir%%||g" \
+        -e "s|%%unix_dir%%||g" \
+        -e "s|%%str_dir%%||g" \
```

Comment:
Similarly, this breaks the script (cf. a 4.14 compilation)

Reply:
Good point, I've fixed it in the same way as above, with the `:?` guard. This makes shellcheck happy and I also tried to configure it on 4.14 - it worked there as well.